maven-java/kalix-maven-plugin/src/main/java/kalix/DeployMojo.java

```diff
@@ -36,32 +35,51 @@ public class DeployMojo extends AbstractMojo {
     @Parameter(property = "kalixContext")
     private String kalixContext;
 
+    @Parameter(property = "kalixProject")
+    private String kalixProject;
+
     @Parameter(defaultValue = "30000", property = "cliTimeoutMs", required = true)
     private Long cliTimeoutMs;
 
     private final Log log = getLog();
 
     /**
-     * We deploy by invoking the services deploy command
+     * We deploy by invoking the services deploy command to 'kalixProject' if set.
+     * If 'kalixProject' is not set then deploying to the currently selected project.
      */
     public void execute() throws MojoExecutionException {
-        log.info("Deploying project to Kalix");
+        final List<String> commandLine;
```

Comment:
Maybe just skip this and pass the args directly to the methods given that it isn't used for anything else?

Reply:
removed